recipes/mmh3/meta.yaml

```
@@ -0,0 +1,37 @@
+{% set name = "mmh3" %}
+{% set version = "2.5.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/hajimes/{{ name|lower }}/archive/v{{ version }}.tar.gz
+  sha256: 2fe4ee15ca09ea48a7bb6493f2a6d2e7659f33af4d46e6b39ff35d77b549e80f
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
```

Please also add...

```suggestion
    - {{ compiler('c') }}
    - {{ compiler('cxx') }}
```

...as the C compiler is also needed when the C++ compiler is added.